nevergrad/optimization/test_optimizerlib.py

```diff
@@ -438,22 +438,20 @@ def _ellips(x: np.ndarray) -> float:
     contextual_budget *= int(max(1, np.sqrt(scale)))
 
     # Let us run the comparison.
-    default = (registry["CMA"] if dimension > 1 else registry["OnePlusOne"])(dimension, contextual_budget, num_workers=num_workers)
-    MetaModel = registry["MetaModel"](dimension, contextual_budget, num_workers=num_workers)
-    default_recom, MetaModel_recom = default.minimize(_target), MetaModel.minimize(_target) 
-    default_data = default_recom.get_standardized_data(reference=default.parametrization)
-    MetaModel_data = MetaModel_recom.get_standardized_data(reference=MetaModel.parametrization)
+    default = registry["CMA" if dimension > 1 else "OnePlusOne"](dimension, contextual_budget, num_workers=num_workers)
+    metamodel = registry["MetaModel"](dimension, contextual_budget, num_workers=num_workers)
+    default_recom, metamodel_recom = default.minimize(_target).value, metamodel.minimize(_target).value 
 
     # Let us assert that MetaModel is better.
-    assert _target(default_data) > _target(MetaModel_data)
+    assert _target(default_recom) > _target(metamodel_recom)
 
     # With large budget, the difference should be significant.
     if budget > 60 * dimension:
-        assert _target(default_data) > 4 * _target(MetaModel_data)
+        assert _target(default_recom) > 4. * _target(metamodel_recom)
 
     # ... even more in the non ellipsoid case.
     if budget > 60 * dimension and not ellipsoid:
-        assert _target(default_data) > 7 * _target(MetaModel_data)
+        assert _target(default_recom) > 7. * _target(metamodel_recom)
```

Comment:
even all this could probably be well factorized, but nevermind for now, it's readable